proto/buf/alpha/registry/v1alpha1/search.proto

```diff
@@ -77,6 +79,21 @@ message CuratedPluginSearchResult {
   bool deprecated = 4;
 }
 
+message ModuleElementSearchResult {
+  string repository_id = 1;
+  string repository_owner = 2;
+  string repository_name = 3;
+  string proto_file_path = 4;
+  string fully_qualified_name = 5;
```

Comment:
I guess with the fully qualified name I can build urls like https://buf.build/acme/petapis/docs/main:pet.v1#pet.v1.PetStoreService, we kinda need to package name as well. Unless all fully qualified names can be transformed into package names easily. I'm not sure that's possible.